query/lambda/lambda_function_multiple_az_configured.sql

```diff
@@ -1,10 +1,18 @@
+
 select
   -- Required Columns
   arn as resource,
   case
-    when vpc_subnet_ids is null then 'alarm'
-    when jsonb_array_length(vpc_subnet_ids) > 1 then 'ok'
+    when vpc_id is null then 'skip'
+    else case    
+      when 
+      (select count(distinct availability_zone_id) from aws_vpc_subnet 
+        where 
+          subnet_id in (select jsonb_array_elements_text(vpc_subnet_ids) )  
+      ) >= 2
+      then 'ok'
     else 'alarm'
+    end
   end as status,
   case
     when vpc_subnet_ids is null then title || ' is not in VPC.'
```

Comment:
@megaproaktiv Can you please also update the case here for the `reason` to match the case above?

Reply:
You mean the indentation? Sure!

Reply:
@megaproaktiv Not necessarily the indentation, but when evaluating the `state`, you updated the check to be `when vpc_id is null then 'skip'`, which seems like a good change. We should perform the same check when evaluating the `reason` as well, so `when vpc_subnet_ids is null then title || ' is not in VPC.'` should probably become `when vpc_id is null then title || ' is not in VPC.'`